USAGE.md

```diff
@@ -35,17 +35,395 @@ Descent 3 Message(Error: Couldn't find the string table.)
-| `-windowed`, `-w`    | boolean | Off                                 | all      | Run game in windowed mode                                         |
+## Base directories
+
+A base directory is a directory that Descent 3 expects game files to be in. When you run Descent 3, it will try to access many different files. Most of those files need to be stored in a base directory. There are two different types of files that are stored in base directories:
+
+- Read-write files are files that can change while you play Descent 3. Examples: `<you name>.plt` and files in the `savegame/` directory.
+- Read-only files are files that do not change while you play Descent 3. Examples: `d3.hog` and files in the `movies/` directory.
+
+Descent 3 has two types of base directories:
+
+- The writable base directory can contain both read-write files and read-only files. There is only one writeable base directory. By default, the writable base directory gets set to the current working directory.
+- The read-only base directories can only contain read-only files. There can be any number of read-only base directories. By default, Descent 3 uses zero read-only base directories.
+
+You can set the writable base directory and the list of read-only base directories using the `-setdir`, `-useexedir` and `-additionaldir` command-line options (see [the next section](#command-line-options)).
+
+When Descent 3 tries to find a read-only file, then it will look through the list of base directories in this order:
+
+- the last read-only base directory that was specified on the command-line,
+- the second-to-last read-only base directory that was specified on the command-line,
+- the third-to-last read-only base directory that was specified on the command-line,
+- …
+- the first read-only base directory that was specified on the command-line and, finally,
+- the writable base directory.
+
+Files that are in base directories that are higher on that list will override files that are in base directories that are lower on that list. For example, lets say that you run Descent 3 like this:
+
+```
+Descent3 -setdir /home/user/my-writable-base-directory -additionaldir /home/user/my-read-only-base-directory
+```
+
+Let’s also say that both `my-writable-base-directory` and `my-read-only-base-directory` contain a file named `d3.hog`. In this example, Descent 3 will load `/home/user/my-read-only-base-directory/d3.hog` because read-only directories have a higher precedence than the writable base directory. Descent 3 will ignore `/home/user/my-writable-base-directory/d3.hog`.
+
+## Command-Line Options
+
+The following command-line options are available in Descent 3. You can set command-line options on the Misc. tab of the Setup section of the Descent 3 launcher or by creating a shortcut to `Descent3.exe`. Case is not significant in command-line options, and `-`, `--`, and `+` are all accepted.
+
+<table>
```

Comment:
Please use markdown-style syntax for table. `USAGE.md` file is included to source tarball, available to users as file and must be easy to read.

Reply:
Do you really think that a Markdown table would be easy to read? I’m looking at [the table in `BUILD.md` right now](https://github.com/DescentDevelopers/Descent3/blob/48c87750f49b8f9cf4a24d3911fb70218e60a0cc/BUILD.md?plain=1#L161-L171), and it’s kind of difficult to read because the lines are over 400 characters long. The table in this PR would be even harder to read because the lines would be even longer. Plus, both of the text editors that I tried using defaulted to using word wrap, so I had to disable word wrap before I could start trying to read the table that was in `BUILD.md`.

Reply:
I agree that HTML formatting for a file we ship in releases is not ideal, it's likely to be opened raw in a text editor. To be honest I'm not fond of md tables either, a good old list `- option: description` is as easy to read on a simple text editor :D

Reply:
@winterheart

> Do you really think that a Markdown table would be easy to read?

Reply:
We can reorder table into plain bullet list like in Linux man pages.

Reply:
> We can reorder table into plain bullet list like in Linux man pages.

OK, I just pushed a new version of this PR that turns the HTML table into a few Markdown bullet lists. How do you think it looks now?